### PR TITLE
Adds a new Conda Feature

### PR DESCRIPTION
**Pull request by samruddhikhandale**
- Adds a new conda feature
- Adds tests

### CODE REVIEW

File: src/conda/NOTES.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Using Conda
+
+This Feature includes [the `conda` package manager](https://docs.conda.io/projects/conda/en/latest/index.html) which is a part of the [Anaconda Distribution](https://repo.anaconda.com). Additional packages installed using Conda will be downloaded from Anaconda or another repository if you configure one. To reconfigure Conda in this container to access an alternative repository, please see information on [configuring Conda channels here](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/channels.html ).
```
Comment (samruddhikhandale) [inline src/conda/NOTES.md:3]: This file is exactly similar to [NOTES.md for anaconda](https://github.com/devcontainers/features/blob/main/src/anaconda/NOTES.md).

I have only added in one sentence --> This Feature includes [the `conda` package manager](https://docs.conda.io/projects/conda/en/latest/index.html) `which is a part of the [Anaconda Distribution](https://repo.anaconda.com)`

cc @Chuxel

Comment (Chuxel) [reply src/conda/NOTES.md:3]: Yep - as discussed, we can add an option to allow the use of something like conda-forge as well. Different people will have different preferences here, so we just need to make sure that people are aware of the nuances.

**Review by edgonmsft — DISMISSED** [review]
LGTM

File: src/conda/NOTES.md (hunk 1 above)
Comment (samruddhikhandale) [reply src/conda/NOTES.md:3]: Sounds good. Added the option `addCondaForge` 👍

**Review by Chuxel — APPROVED** [review]

**Review by edgonmsft — APPROVED** [review]
LGTM

**Comment by eitsupi** [comment]
FYI, as an alternative to this for those who prefer to use mamba instead of conda, the following feature is now available.
https://github.com/rocker-org/devcontainer-features/tree/main/src/miniforge

**Pull request merged**